setup.py

```diff
@@ -18,6 +18,7 @@
         'pillow>=5.4.1',
 
         'quilt==2.9.15',
+        'quilt3==3.1.0',
```

Comment:
Ah! That's nice! So we can keep using both versions of `quilt` until we completely transition to `version 3`?

Reply:
Yep!